tests/foreman/cli/test_user.py

```diff
@@ -675,6 +674,36 @@ def test_negative_create_user_4(self, email):
         self.assertNotEqual(result.return_code, 0)
         self.assertTrue(result.stderr)
 
+    @skip_if_bug_open('bugzilla', 1204686)
+    @data(
+        ''
+    )
+    def test_bugzilla_1204686(self, email):
+        """@Test: Create User with invalid Email Address
+
+        @Feature: User - Negative Create
+
+        @Steps:
+        1. Create User for all invalid Email Address in [2]
```

Comment:
This test is just using one data entry, then the `@data` decorator is not needed you can use the `''` string in the test body. Like:

```
options = {
    # ...
    'mail': '',
    # ...
}